labs/CanvasLayout/samples/CanvasLayout.Uwp/CanvasLayout.Uwp.csproj

```diff
@@ -35,9 +35,6 @@
     <AssemblyName>CanvasLayoutExperiment.Samples.Uwp</AssemblyName>
     <ProjectGuid>{C3F460EC-A0A6-4B22-A91A-6CF98B30FD8A}</ProjectGuid>
   </PropertyGroup>
-  <ItemGroup>
-    <Compile Include="Properties\AssemblyInfo.cs" />
```

Comment:
Need to file a tech debt ticket for this just before we close off. New experiments still in PR will need to get this change.

Reply:
Yeah, if they update they'll get duplicate metadata things, so should be pretty obvious.

Reply:
Looks like there's only 2 open experiment PRs. Now that this is merged to main, I'll update those branches to latest main and fix this, before they do.